src/db/index.ts

```diff
@@ -4,12 +4,28 @@ import { Database } from 'bun:sqlite';
 import path from 'path';
 import * as schema from './schema';
 
-const sqlite = new Database(
-  path.join(import.meta.dir, '../../data/lgtbot.db'),
-  {
-    create: true,
-  }
-);
+const sqlite =
+  process.env.NODE_ENV === 'test'
+    ? new Database(':memory:')
+    : new Database(path.join(import.meta.dir, '../../data/lgtbot.db'), {
+        create: true,
+      });
+
+if (process.env.NODE_ENV === 'test') {
+  sqlite.run(`
+    CREATE TABLE IF NOT EXISTS book_club_bans (
+      id INTEGER PRIMARY KEY AUTOINCREMENT,
+      discord_user_id TEXT NOT NULL,
+      discord_message_ids TEXT NOT NULL
+    );
+
+    CREATE TABLE IF NOT EXISTS twitch_subscriptions (
+      id INTEGER PRIMARY KEY AUTOINCREMENT,
+      username TEXT NOT NULL,
+      twitch_subscription_id TEXT NOT NULL
+    );
+  `);
+}
```

Comment:
There's probably a better way to do these migrations but I'll figure it out later